.github/workflows/push.yml

```diff
@@ -0,0 +1,34 @@
+name: Push
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
If we do a patch release, which usually happens on a release branch (like https://github.com/vmware-tanzu/velero/tree/release-1.3), I assume this won't work.

Can we use a wild card for `release-*`, too?